src/lib/skilling/skills/thieving/stealables.ts

```diff
@@ -0,0 +1,400 @@
+	},
+	{
+		name: 'Cave goblin',
+		level: 36,
+		xp: 40,
+		table: CaveGoblin.pickpocketTable!,
+		id: CaveGoblin.id,
+		stunTime: 5,
+		stunDamage: 1,
+		// No current data on slope/intercept
+		slope: 0.66879,
+		intercept: 27.92979
+	},
+	{
+		name: 'Master Farmer',
+		level: 38,
+		xp: 43,
+		table: MasterFarmer.pickpocketTable!,
+		id: MasterFarmer.id,
+		stunTime: 5,
+		stunDamage: 3,
+		slope: 0.59786,
+		intercept: 34.95186
+	},
+	{
+		name: 'Guard',
+		level: 40,
+		xp: 46.8,
+		table: Guard.pickpocketTable!,
+		id: Guard.id,
+		stunTime: 5,
+		stunDamage: 2,
+		slope: 0.76776,
+		intercept: 18.13176
+	},
+	{
+		name: 'Fremennik Citizen',
+		level: 45,
+		xp: 65,
+		table: FremennikCitizen.pickpocketTable!,
+		id: FremennikCitizen.id,
+		stunTime: 5,
+		stunDamage: 2,
+		// No current data on slope/intercept
+		slope: 0.76776,
+		intercept: 18.13176
+	},
+	{
+		name: 'Desert Bandit',
+		level: 53,
+		xp: 79.5,
+		table: DesertBandit.pickpocketTable!,
+		id: DesertBandit.id,
+		stunTime: 5,
+		stunDamage: 3,
+		slope: 0.75735,
+		intercept: 19.16235
+	},
+	{
+		name: 'Knight of Ardougne',
+		level: 55,
+		xp: 84.3,
+		table: KnightOfArdougne.pickpocketTable!,
+		id: KnightOfArdougne.id,
+		stunTime: 5,
+		stunDamage: 3,
+		slope: 0.77209,
+		intercept: 17.70309
+	},
+	{
+		name: 'Pollnivian Bandit',
+		level: 55,
+		xp: 84.3,
+		table: PollnivnianBandit.pickpocketTable!,
+		id: PollnivnianBandit.id,
+		stunTime: 5,
+		stunDamage: 5,
+		// No current data on slope/intercept
+		slope: 0.77209,
+		intercept: 17.70309,
+		customTickRate: 2.4
+	},
+	{
+		name: 'Yanille Watchman',
+		level: 65,
+		xp: 137.5,
+		table: YanilleWatchman.pickpocketTable!,
+		id: YanilleWatchman.id,
+		stunTime: 5,
+		stunDamage: 3,
+		slope: 0.58,
+		intercept: 5.47
+	},
+	{
+		name: 'Menaphite Thug',
+		level: 65,
+		xp: 137.5,
+		table: MenaphiteThug.pickpocketTable!,
+		id: MenaphiteThug.id,
+		stunTime: 5,
+		stunDamage: 5,
+		slope: 0.65485,
+		intercept: 29.30985,
+		customTickRate: 2.4
+	},
+	{
+		name: 'Paladin',
+		level: 70,
+		xp: 151.7,
+		table: Paladin.pickpocketTable!,
+		id: Paladin.id,
+		stunTime: 5,
+		stunDamage: 3,
+		slope: 0.40429,
+		intercept: 18.95529
+	},
+	{
+		name: 'Gnome',
+		level: 75,
+		xp: 198.5,
+		table: Gnome.pickpocketTable!,
+		id: Gnome.id,
+		stunTime: 5,
+		stunDamage: 1,
+		slope: 0.47565,
+		intercept: 0.18065
+	},
+	{
+		name: 'Hero',
+		level: 80,
+		xp: 275,
+		table: Hero.pickpocketTable!,
+		id: Hero.id,
+		stunTime: 6,
+		stunDamage: 4,
+		slope: 0.39056,
+		intercept: 0.78456
+	},
+	{
+		name: 'Vyre',
+		level: 82,
+		xp: 306.9,
+		table: Vyre.pickpocketTable!,
+		id: Vyre.id,
+		stunTime: 5,
+		stunDamage: 5,
+		slope: 0.48813,
+		intercept: 2.06513
+	},
+	{
+		name: 'Elf',
+		level: 85,
+		xp: 353,
+		table: PrifddinasElf.pickpocketTable!,
+		id: PrifddinasElf.id,
+		stunTime: 6,
+		stunDamage: 5,
+		slope: 0.42077,
+		intercept: -2.20623,
+		qpRequired: 200
+	},
+	{
+		name: 'TzHaar-Hur',
```

Comment:
these are locked behind having a firecape in the real game, not sure if you want to try to do that here too